pgtap/chinese/chinesePostman/chinesePostman-rows-check.sql

```diff
@@ -30,11 +30,15 @@ SELECT * FROM pgr_chinesePostman(
     ORDER BY RANDOM()'
 );
 
+SELECT todo_start('Fix the code to return same set of rows');
```

Comment:
So lets refine:

SELECT todo_start('Chinese Postman Problem is Experimental, tests left pending');